.gitignore

```diff
@@ -115,9 +115,9 @@ open-metadata-implementation/user-interfaces/presentation-server/nodejs/yarn.loc
 open-metadata-implementation/user-interfaces/presentation-server/nodejs/node/**
 open-metadata-implementation/user-interfaces/presentation-server/nodejs/.env
 
-open-metadata-implementation/user-interfaces/presentation-server/cra-client/node_modules
+open-metadata-implementation/user-interfaces/presentation-server/cra-client/node_modules/**
```

Comment:
You need to create equivalents of the other excludes from the existing presentation server .gitignore otherwise these files will end up getting into git. 
dist/**        if it has its own output folder.
package-lock.json
.vscode
.log
yarn.lock

If any other files could be generated in the new build, then these need to be ignored as well.